src/app/action-items/action-items.component.ts

```diff
@@ -1,19 +1,22 @@
+import { ActionTasksElement } from '../services/tasksService/tasks.service';
 import { Component, OnInit, OnDestroy } from '@angular/core';
 import { map } from 'rxjs/operators';
-import { TasksService } from '../services/tasks.service';
-import { DaysLeftToDeadlineService } from '../services/days-left-to-deadline.service';
+import { TasksService } from '../services/tasksService/tasks.service';
+import { DaysLeftToDeadlineService } from '../services/daysLeftToDeadlineService/days-left-to-deadline.service';
 
 @Component({
   selector: 'app-action-items',
   templateUrl: './action-items.component.html',
   styleUrls: ['./action-items.component.css']
 })
 export class ActionItemsComponent implements OnInit {
-  dataSource;
+  dataSource: ActionTasksElement[];
   loading = true;
-  displayedColumns: string[] = ['title', 'type', 'completed', 'dueDate', 'daysLeft'];
   constructor(private tasksService: TasksService, private daysCountService: DaysLeftToDeadlineService) {}
   ngOnInit() {
+    this.getTasks();
+  }
+  getTasks() {
```

Comment:
This method shouldn't be exposed. We should not change the public class definition for the test. Especially that this method is used only once. It might be a separate method if you prefer, but this should be left private.